src/phase0/mod.rs

```diff
@@ -29,9 +29,10 @@ pub use state_transition::*;
 pub use validator::*;
 
 pub const BASE_REWARDS_PER_EPOCH: u64 = 4;
-pub const DEPOSIT_CONTRACT_TREE_DEPTH: usize = 2usize.pow(5);
+pub const DEPOSIT_CONTRACT_TREE_DEPTH: usize = 2usize.saturating_pow(5);
 pub const JUSTIFICATION_BITS_LENGTH: usize = 4;
-pub const DEPOSIT_DATA_LIST_BOUND: usize = 2usize.pow(DEPOSIT_CONTRACT_TREE_DEPTH as u32);
+pub const DEPOSIT_DATA_LIST_BOUND: usize =
+    2usize.saturating_pow(DEPOSIT_CONTRACT_TREE_DEPTH as u32);
```

Comment:
if we can't just use `pow` then we can just write out the integer as `usize`

Reply:
Yh! so this is due to the compilation target `wasm32-unknown-unknown`
But if i do 2^32 which is 4294967296, I am going to end up with an overflowing literal